irods_specific_implementation/irods_rest_api_base.h

```diff
@@ -232,7 +232,7 @@ namespace irods::rest
 
         std::string decode_url(const std::string& _in) const
         {
-            trace("Decoding input [{}] ...", _in);
+            trace("Decoding input [{}] ...", _in); // TODO This prints incoming arguments! (e.g. Passwords!)
```

Comment:
oh.   hmmm...  yeah i don't think we should have this here.   debugging is easy enough - consider commenting it out, so developers can easily turn it back on?

Reply:
left comment explaining why it is commented out.